samza-core/src/main/scala/org/apache/samza/util/CoordinatorStreamUtil.scala

```diff
@@ -48,6 +48,23 @@ object CoordinatorStreamUtil extends Logging {
     new MapConfig(coordinatorSystemConfig)
   }
 
+  /**
+   * Creates coordinator stream from config.
+   *
+   * @param config to create coordinator stream.
+   */
+  def createCoordinatorStream(config: Config): Unit = {
+    val systemAdmins = new SystemAdmins(config)
+
+    // Create the coordinator stream if it doesn't exist
+    info("Creating coordinator stream")
```

Comment:
This log is going repeat itself for every attempt we do this. 
Can you update the log to reflect its an attempt or remove it if there is logging within the creation flow that differentiates between actually creating the resource vs no-op?

Reply:
It is already there, createCoordinatorStream(config: Config) encapsulate another createCoordinatorStream(coordinatorSystemStream, coordinatorSystemAdmin) where there is a logging to distinguish actual creation or no-op.

Reply:
then i'd prefer to remove this redundant logging. do you have anything else in mind on why this is needed then?

Reply:
Not really, this is the same logic I copy paste from the previous CoordinatorStreamUtil#writeConfigToCoordinatorStream.